course-v2/assets/js/mobile_course_drawers.ts

```diff
@@ -0,0 +1,35 @@
+export const initCourseDrawersClosingViaSwiping = () => {
+  elementSwiping("mobile-course-nav", "mobile-course-nav-toggle", "L")
+  elementSwiping("course-info-drawer", "mobile-course-info-toggle", "R")
+}
+
+/**
+ * It adds touch event listener to the element and clicks the button when swiped in the mentioned direction.
+ *
+ * @param {string} elementId element on which touch eventlistenter is added.
+ * @param {string} buttonId This button will be clicked on swiping
+ * @param {string} swipeDirection L= Swipe left, R= Swipe Right
+ */
+const elementSwiping = (
+  elementId: string,
+  buttonId: string,
+  swipeDirection: string
```

Comment:
You can also make swipeDirection an Enum or a Union Type with values of left and right. which would make it more readable and a bit more error safe.